frontends/ol-components/src/components/Card/Card.tsx

```diff
@@ -130,17 +139,72 @@ const Bottom = styled.div`
 const Actions = styled.div`
   display: flex;
   gap: 8px;
-  position: absolute;
-  bottom: 16px;
-  right: 16px;
 `
 
+/**
+ * Click the child anchor element if the click event target is not the anchor itself.
+ *
+ * Allows making a whole region clickable as a link, even if the link is not the
+ * direct target of the click event.
+ */
+export const useClickChildHref = (
+  href?: string,
+  onClick?: React.MouseEventHandler<HTMLElement>,
+): React.MouseEventHandler<HTMLElement> => {
+  return useCallback(
+    (e) => {
+      onClick?.(e)
+      if (!e.currentTarget.contains(e.target as Node)) {
+        // This happens if click target is a child in React tree but not DOM tree
+        // This can happen with portals.
+        // In such cases, data-card-actions won't be a parent of the target.
+        return
+      }
+      const anchor = e.currentTarget.querySelector<HTMLAnchorElement>(
+        `a[href="${href}"]`,
+      )
+      const target = e.target as HTMLElement
+      if (!anchor || target.closest("a, button, [data-card-action]")) return
+      if (e.metaKey || e.ctrlKey) {
+        const opts: PointerEventInit = {
+          bubbles: false,
+          metaKey: e.metaKey,
+          ctrlKey: e.ctrlKey,
+        }
+        anchor.dispatchEvent(new PointerEvent("click", opts))
+      } else {
+        anchor.click()
+      }
+    },
+    [href, onClick],
+  )
+}
+
 type CardProps = {
   children: ReactNode[] | ReactNode
   className?: string
   size?: Size
+  /**
+   * If provided, the card will render its title as a link.
+   *
+   * Clicks on the entire card can be forwarded to the link via `forwardClicksToLink`.
+   */
   href?: string
-}
+  /**
+   * Defaults to `false`. If `true`, clicking the whole card will click the
+   * href link as well.
+   *
+   * NOTES:
+   *  - If using Card.Content to customize, you must ensure the content includes
+   *  an anchor with the card's href.
+   *  - Clicks will NOT be forwarded if it is (or is a child of):
+   *    - an anchor or button element
+   *    - OR an element with data-card-action
+   */
+  forwardClicksToLink?: boolean
```

Comment:
Originally, I made this the default behavior. However, since reliably NOT triggering the link when clicking interactive children requires some effort from consumer (adding `data-card-action` if the interactive child is not a anchor or button) I decided to make it opt-in to increase visibility.